pygeoapi/templates/landing_page.html

```diff
@@ -43,12 +43,14 @@ <h1>{{ config['metadata']['identification']['title'] }}</h1>
       </div>
     </div>
   </section>
+  {% if data['collection'] %}
```

Comment:
@IngenieroGeomatico there is no attribute data['collection], so you need to find a different way of checking if collections other than stac and process exist. Right now, we are never entering this condition.

Reply:
Here is the data structure (in a server with features and maps collections):

`      },
      {
         "rel":"service-desc",
         "type":"application/vnd.oai.openapi+json;version=3.0",
         "title":"The OpenAPI definition as JSON",
         "href":"http://localhost:5000/openapi"
      },
      {
         "rel":"service-doc",
         "type":"text/html",
         "title":"The OpenAPI definition as HTML",
         "href":"http://localhost:5000/openapi?f=html",
         "hreflang":"Locale(""en",
         "territory=""US"")"
      },
      {
         "rel":"conformance",
         "type":"application/json",
         "title":"Conformance",
         "href":"http://localhost:5000/conformance"
      },
      {
         "rel":"data",
         "type":"application/json",
         "title":"Collections",
         "href":"http://localhost:5000/collections"
      },
      {
         "rel":"http://www.opengis.net/def/rel/ogc/1.0/processes",
         "type":"application/json",
         "title":"Processes",
         "href":"http://localhost:5000/processes"
      },
      {
         "rel":"http://www.opengis.net/def/rel/ogc/1.0/job-list",
         "type":"application/json",
         "title":"Jobs",
         "href":"http://localhost:5000/jobs"
      }
   ],
   "title":"pygeoapi default instance",
   "description":"pygeoapi provides an API to geospatial data",
   "processes":true,
   "stac":true
}
`

Reply:
Hi!

I have seen two possible solutions:

1./ check in config["resources"] if exist any "collections"

2./ add to data "collections" key with true/false value  (75 / 5.000
Resultados de traducción
Resultado de traducción
I like this one better so that it maintains the same philosophy as processes and stac)


@doublebyte1 ¿What do you think is better?

Reply:
@IngenieroGeomatico I agree - option 2 is a neat solution! ThankYou 😄 

I think that the errors that you are getting on the unit tests are unrelated to these changes.